ocp_resources/volume_snapshot_content.py

```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import Resource
+
+
+class VolumeSnapshotContent(Resource):
```

Comment:
Please add to_dict method with the required values to create  VolumeSnapshotContent

Reply:
VolumeSnapshotContent is created by the snapshot common controller, we don't create it ourselves.

Reply:
according to https://kubernetes.io/docs/concepts/storage/volume-snapshots/#static
`A cluster administrator creates a number of VolumeSnapshotContents`
Please add to_dict

Reply:
Right, it's about static provisioning, but we always prefer it to be dynamically provisioned
https://kubernetes.io/docs/concepts/storage/volume-snapshots/#volume-snapshot-contents
`In dynamic provisioning, the snapshot common controller creates VolumeSnapshotContent objects. `

Reply:
it does not matter; this is a resource that can be manually created and the code needs to support it. when using it in your implementation, you can select how to use it.